nexus/db-queries/src/db/datastore/deployment.rs

```diff
@@ -412,6 +554,7 @@ impl DataStore {
         Ok(Blueprint {
             id: blueprint_id,
             blueprint_zones,
+            blueprint_disks,
             parent_blueprint_id,
             internal_dns_version,
```

Comment:
github won't let me put a comment on something that isn't in the diff, but `blueprint_delete` also needs to be updated to remove the disks associated with the blueprint-being-deleted.